python/lsst/daf/butler/core/formatter.py

```diff
@@ -73,7 +73,7 @@ class Formatter(metaclass=ABCMeta):
     """
 
     # Should we use ... to say that the parameters are unconstrained?
-    supportedWriteParameters: ClassVar[Optional[Union[FrozenSet[str], Set[str]]]] = frozenset()
+    supportedWriteParameters: ClassVar[Optional[Union[FrozenSet[str], Set[str]]]] = None
```

Comment:
Might want to just use `MutableSet[str]` rather than the `Union` for this type.

Reply:
FrozenSet isn't mutable though???

Reply:
Oh, right.  It would need to be `AbstractSet[str]`.  But that's probably what you want - mypy might complain later when you try to use mutators, but it would also complain similarly about the union.